solver/src/solver/http_solver/settlement.rs

```diff
@@ -239,7 +203,10 @@ mod tests {
             exec_sell_amount: 7.into(),
         };
         let updated_uniswap = UpdatedAmmModel {
-            updates: hashmap! { t0 => 8, t1 => -9 },
```

Comment:
I interpreted this as t1 is sell token (because of negative 9 here). Maybe I was incorrect in my assumption.

Reply:
Double checked and confirmed with @marcovc.